make/DynamoRIOConfig.cmake.in

```diff
@@ -178,7 +178,10 @@
 #  set(DynamoRIO_VISATT ON)
 #
 # To request the old REG_ enum symbols (which were changed to DR_REG_ to
-# avoid conflicts with system headers) set this variable:
+# avoid conflicts with system headers) set this variable (newly added AVX-512
+# registers DR_REG_ZMM0 - DR_REG_ZMM31 as well as DR_REG_K0 - DR_REG_K7 are
+# excluded from this):
```

Comment:
This is confusing to me: I can't tell whether "excluded from this" means the REG_ symbols for them are always available whether I set this compat param or not.  I would suggest a separate sentence that clarifies rather than the current parenthetical which seems ambiguous.

Reply:
Done / re-phrased.